lib/plugins/create/templates/aws-csharp/obj/Debug/netcoreapp2.1/aws-csharp.AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------
```

Comment:
I think this file is not expected to be included in a PR

Reply:
@medikoo, I've removed those files and pushed the changes. I did a rebase to prevent the extra commits from showing in the history, not sure if you care or not.

Also, the build is failing, but it is unrelated to the changes I've made.